oap-server/server-bootstrap/src/main/resources/envoy-metrics-rules/envoy.yaml

```diff
@@ -61,17 +61,17 @@ metricsRules:
     # envoy_cluster_metrics
   - name: cluster_membership_healthy
     exp: envoy_cluster_metrics.tagMatch('metrics_name' , '.+membership_healthy').tagMatch('metrics_name' , 'cluster.outbound.+|cluster.inbound.+').tagNotMatch('cluster_name' , '.+kube-system').sum(['app', 'instance' , 'cluster_name'])
-  - name: cluster_upstream_cx_active
```

Comment:
Whether the metric name can be consistent with the original before?

Reply:
some prefix +  metric names too long could cause some storage issues. I refactor them and keep consistently